requirements/dev.txt

```diff
@@ -41,66 +40,59 @@ edx-opaque-keys==2.0.0    # via edx-drf-extensions
 enum34==1.1.6
 filelock==3.0.12          # via tox
 fs==2.4.11
-funcsigs==1.0.2           # via mock
```

Comment:
Some of these removed dependencies still seem to be needed; did you run `make upgrade` under Python 2.7, and is the `--rebuild` option being passed to pip-compile?

Reply:
I was running make upgrade in edx-platform, I reran things and fixed this